lib/workload/orcabus-stateless-stack.ts

```diff
@@ -12,7 +15,26 @@ export interface OrcaBusStatelessConfig {
   lambdaRuntimePythonVersion: aws_lambda.Runtime;
   bclConvertFunctionName: string;
   rdsMasterSecretName: string;
-  eventSourceQueueName?: string;
+  filemanagerDependencies?: FilemanagerDependencies;
+}
+
+export interface FilemanagerDependencies {
+  /**
+   * Queue name used by the EventSource construct.
+   */
+  eventSourceQueueName: string;
+  /**
+   * Buckets defined by the EventSource construct.
+   */
+  eventSourceBuckets: string[];
+  /**
+   * Database secret name for the filemanager.
+   */
+  databaseSecretName: string;
+  /**
+   * Database security group name to allow the filemanager Lambda to connect.
+   */
+  databaseSecurityGroupName: string;
```

Comment:
Could you reuse `lambdaSecurityGroupName` from base stateless config? That was the intent as shared SG.